package.json

```diff
@@ -31,13 +31,13 @@
     "inputmask-core": "^2.1.1"
   },
   "peerDependencies": {
-    "react": "0.14.x"
+    "react": "15.x.x"
```

Comment:
Thanks, @szyablitsky and @iamdustan! Good points. I like the idea of `0.14.x || 15.0.x`; it doesn't allow major or minor updates, which could introduce breaking changes, and communicates at a glance which versions are supported. Updated!